lib/controllers/status-bar-tile-controller.js

```diff
@@ -251,18 +262,28 @@ export default class StatusBarTileController extends React.Component {
   async pull() {
     await this.attemptGitOperation(
       () => this.doPull(),
-      description => {
-        if (/error: Your local changes to the following files would be overwritten by merge/.test(description)) {
-          const lines = description.split('\n');
+      error => {
+        if (/error: Your local changes to the following files would be overwritten by merge/.test(error.stdErr)) {
+          const lines = error.stdErr.split('\n');
           const files = lines.slice(3, lines.length - 3).map(l => `\`${l.trim()}\``).join('<br>');
           return {
+            type: 'Error',
             message: 'Pull aborted',
             description: 'Local changes to the following would be overwritten by merge:<br>' + files +
               '<br>Please commit your changes or stash them before you merge.',
           };
+        } else if (/Automatic merge failed; fix conflicts and then commit the result./.test(error.stdOut)) {
+          this.props.ensureGitTabVisible();
+          return {
+            type: 'Warning',
+            message: 'Your pull resulted in merge conflicts',
+            description: `Your last pull resulted in merge conflicts between your local changes
+              and the changes on the remote. You can resolve the conflicts with the Git panel
+              and commit them to continue.`,
```

Comment:
I actually brevity makes more sense for a notification. Easier to scan, and not feel overwhelmed by text. Thanks for the great suggestions :)